tests/migration/ember-addon/steps/move-addon-files/other-files.test.js

```diff
@@ -0,0 +1,40 @@
+import { moveAddonFiles } from '../../../../../src/migration/ember-addon/steps/move-addon-files.js';
+import {
+  augmentedOptions,
+  options,
+} from '../../../../helpers/shared-test-setups/typescript.js';
+import {
+  assertFixture,
+  loadFixture,
+  test,
+} from '../../../../helpers/testing.js';
+
+test('migration | ember-addon | steps | move-addon-files > other files', function () {
+  const inputProject = {
+    'addon-test-support': {
+      '.gitkeep': '',
+    },
+    blueprints: {
+      '.gitkeep': '',
+    },
+  };
```

Comment:
This may be a temporary solution to asserting that the codemod does consider `addon-test-support` and `blueprints`. At the moment, I don't have a test fixture with either of these folders.